docs/howto/how-to-make-api-tool/api-tool-guide.md

```diff
@@ -0,0 +1,250 @@
+            pa.field("fga", pa.int64()),
+            pa.field("fgm", pa.int64()),
+            pa.field("fg3", pa.int64()),
+            pa.field("fta", pa.int64()),
+            pa.field("ftm", pa.int64()),
+            pa.field("turnovers", pa.int64()),
+        ])
+        assists = []
+        blocks = []
+        rebounds = []
+        minutes = []
+        points = []
+        steals = []
+        fga = []
+        fgm = []
+        fg3 = []
+        fta = []
+        ftm = []
+        tos = []
+        for stat in stats_json:
+            assists.append(stat["ast"])
+            blocks.append(stat["blk"])
+            rebounds.append(stat["reb"])
+            mins, secs = stat["min"].split(":")
+            mins_in_secs = int(mins)*60 + int(secs)
+            minutes.append(mins_in_secs)
+            points.append(stat["pts"])
+            steals.append(stat["stl"])
+            fga.append(stat["fga"])
+            fgm.append(stat["fgm"])
+            fg3.append(stat["fg3m"])
+            fta.append(stat["fta"])
+            ftm.append(stat["ftm"])
+            tos.append(stat["turnover"])
+        
+        table = pa.table(
+            [points, rebounds, assists, steals, blocks, minutes, fga, fgm, fg3, fta, ftm, tos],
+            schema=stats_chema
+        )
+        return table
+```
+This function iterates over the returned JSON from step 1 and pulls the relevant fields into an array of arrays, which conforms to the [`pa.schema`](https://arrow.apache.org/docs/python/generated/pyarrow.Schema.html) that we defined in the beginning of the function.
+
+> Other times, the data comes in a better format and you can convert to Apache Arrow format more easily with the [built-in helper function](https://arrow.apache.org/docs/python/json.html) 
+### 3. Getting the min, max, and mean 
+Now that we have a `pyarrow.Table` representation of the JSON data, we can use the built-in [compute functions](https://arrow.apache.org/docs/python/compute.html) to calculate the min, max, and mean of each of our statistical categories with the following function:
+
+```python
+    def compute_aggregate_output(table):
+        mins = []
+        maxs = []
+        means = []
+        cat = []
+        for stat, col_name in zip(table, table.column_names):
+            if stat.type == pa.duration('s'):
+                stat = stat.cast(pa.int64())
+            min_max = pc.min_max(stat)
+            mins.append(min_max[0].as_py())
+            maxs.append(min_max[1].as_py())
+            means.append(pc.mean(stat).as_py())
+            cat.append(col_name)
+        
+        aggregate_tables = [cat, means, mins, maxs]
+
+        return pa.table(aggregate_tables, names = ["category", "mean", "min", "max", ])
+```
+
+This function will call the compute function `pc.min_max()` and `pc.mean()` on each statistical category of our `pa.Table` that we created in step 2. It then aggregates all of these into a new table and returns it.
```

Comment:
TW review - update to:

This function calls the compute function `pc.min_max()` and `pc.mean()` on each statistical category of the `pa.Table` that we created in step 2. It then aggregates all of these into a new table and returns that table.